src/animators/animator.ts

```diff
@@ -1,4 +1,39 @@
 module Plottable {
+export module Animators {
+  export type d3SelectionOrTransition = d3.Selection<any>|d3.Transition<any>;
```

Comment:
I think Typescript is slowly moving away from having the union types be bunched up and it also looks a bit cluttered.

Can you add a whitespace around `|`